packages/map/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v5.0.0 Breaking changes
```

Comment:
why is it a breaking change?

Reply:
You don't really need to change anything from user perspective but still, we are migrating from one library to another

Reply:
If users don't need to do anything, then it's no breaking change

Reply:
Hm, I already published the package would be difficult to revert, let's have this version

Reply:
😭

Reply:
no problem just bump the dependencies in the apps

Reply:
At least add a comment here in CHANGELOG explaining that nothing needs to be changed